bitnami/postgresql-repmgr/11/debian-11/rootfs/opt/bitnami/scripts/libpostgresql.sh

```diff
@@ -368,6 +368,19 @@ postgresql_create_replication_user() {
     echo "CREATE ROLE \"$POSTGRESQL_REPLICATION_USER\" REPLICATION LOGIN ENCRYPTED PASSWORD '$escaped_password'" | postgresql_execute
 }
 
+########################
+# Change postgresql.conf by setting archive_command value
+# Globals:
+#   POSTGRESQL_*
+# Arguments:
+#   None
+# Returns:
+#   None
+#########################
+postgresql_configure_archive_command() {
+  info "Overriding archive_command default value..."
+  postgresql_set_property "archive_command" "cp %p ${POSTGRESQL_WAL_ARCHIVE_COMMAND_DIR}/%f"
```

Comment:
Should the `POSTGRESQL_WAL_ARCHIVE_COMMAND_DIR` be persisted? Should we place it under `/bitnami/postgresql` ?

Reply:
In that case we should ensure that the directory exists and has the right permissions. Please take a look to this piece of code: https://github.com/bitnami/containers/blob/934b1fa547a3bf4cf506cb48a185dd236def47b3/bitnami/postgresql-repmgr/11/debian-11/rootfs/opt/bitnami/scripts/libpostgresql.sh#L625-L629

NOTE: In your case the directory can be empty